vagrant-post-script/prerequisites.sh

```diff
@@ -3,8 +3,8 @@
 sudo apt-get update > /dev/null #avoid spamming the output
 sudo apt-get install curl htop -y
 
-NODE_VERSION=16
-NPM_VERSION=9.6.4
+NODE_VERSION=18
+NPM_VERSION=9.8.8
```

Comment:
```suggestion
NPM_VERSION=9.8.0
```
This is the latest version of NPM